### DISCUSSION TITLE
next/link <Link /> accessibility concerns

### DISCUSSION THREAD

**Discussion by phc5** [vercel/next.js#13125] [Help]
Hello,

I have a couple of links on the header of my personal website that use `<Link />` for navigation. I tried tabbing through my website to see what came into focus and links that used `<Link />` weren't focused. 

I made a fix to pass `passHref` to these `<Link />`s and now they come into focus on tab.

Is there a reason the `passHref` prop on <Link /> is not `true` by default? 

I can propose a couple of solutions: 

1. Default to `true` to better conform to accessibility standards + update documentation.
-or-
2. Update the documentation to pass `passHref` as true in examples that use `<Link />`. [https://nextjs.org/docs/api-reference/next/link](https://nextjs.org/docs/api-reference/next/link)

If any of these options sound good, I'd love to work on it. 
Thanks

**Comment by bplittle** [comment]
Perhaps it's not the default because there are cases where the immediate child is not an `<a>` element. We ran into this once.
HOWEVER, I would still prefer it was the default, and that the case I ran into was an exception. Accessibility functionality should be default.

**Comment by leerob** [comment]
https://github.com/vercel/next.js/issues/23713

**Comment by borispoehland** [comment]
@leerob has the new Link accessibility problems as well? I can't manage to get it to focus

**Reply by devlato** [reply]
+1 focus doesn't work as expected

**Reply by leerob** [reply]
Could you create a new issue with a reproduction and we can take a look?
